src/styles/index.scss

```diff
@@ -1,3 +1,66 @@
+@import './utils/variable';
+@import './utils/mixin';
+
 body {
   margin: 0;
+  height: 100vh;
+
+  @include centered-content;
+}
+
+.stopwatch {
+  height: $stopwatch-size;
+  width: $stopwatch-size;
+  border: 1vmin dotted $stopwatch-border-color;
+  border-radius: $border-radius;
+  position: relative;
+
+  @include centered-content;
+
+  &__hand {
+    position: absolute;
+    transform-origin: bottom;
+    bottom: 50%;
+
+    @include centered-content;
+
+    &--min {
+      height: $minutes-hand-height;
```

Comment:
The variable '$minutes-hand-height' is used here. Ensure it is defined in the './utils/variable' file or elsewhere in your project.